content/en/blog/_posts/2021-12-16-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes is moving on from Dockershim: commitments and next steps"
+date: 2021-12-16
```

Comment:
Please update this (and the path) to publish on either:
2021-12-20 (preferred)

2021-12-22
2021-12-19

Reply:
The motivation is to separate it from the Kubernetes post-release announcements for v1.23

Reply:
@sftim - another option is publish this January 3rd, 2021, and kick off the new year with it. Thoughts?

Reply:
3rd of January SGTM. We can schedule it all in ahead of time.
I'm very happy to hear any official thoughts from CNCF on timing, too (if different).